ui/selectors/assets.ts

```diff
@@ -200,3 +200,67 @@ export const getMultiChainAssets = createDeepEqualSelector(
     });
   },
 );
+
+const zeroBalanceAssetFallback = { amount: 0, unit: '' };
+
+export const getMultichainAggregatedBalance = createDeepEqualSelector(
```

Comment:
In a different PR, can we add some selector and selector equality tests?

- Test that a selector is returning the same data
- (imo more important) that a selector is returning the same identical data with the same inputs. `expect(x).toBe(y)` (not `.toEqual`).

A lot of re-renders when investigating turn out to be bad selectors, which because they return new data, causes renders on every redux state change.

Reply:
We are okay here since we are specifying deep-equal selector here.

But I think we could also get away with a `createSelector` if the returning data is a primitive type (e.g. number)